README.md

```diff
@@ -57,7 +58,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: **O(n^2)**
```

Comment:
I oversimplified! (I had thought that during the lecture, it had been suggested to generalize `O(n * m)` cases as `O(n^2)`.) I believe this is `O(n * m)` when not oversimplified because the two arrays are different and wouldn't necessarily have the same length if changed, etc. Is that right?